files/fluentd/run.sh

```diff
@@ -68,8 +67,31 @@ fi
 # Determine allowed total given the number of outputs we have.
 ALLOWED_DF_LIMIT=$(expr $DF_LIMIT \* $ALLOWED_PERCENT_OF_DISK / 100) || :
 
+TOTAL_LIMIT_SIZE_CONF=$(grep "total_limit_size" $FLUENT_CONF |  head -1 | awk '{print $2}' )
+
+if [[ $TOTAL_LIMIT_SIZE_CONF == *"TOTAL_LIMIT_SIZE"* ]]; then
```

Comment:
I'll defer to bash experts here.  not sure what this block is doing other then converting to bytes in the else block

Reply:
I observed fluent.conf has ` total_limit_size "#{ENV['TOTAL_LIMIT_SIZE'] ||  8589934592 }" #8G`  when CLO starts so if block just ignores it if it finds `TOTAL_LIMIT_SIZE` in the line. Once user changes this value using ClusterLogging instance `total_limit_size` set to actual value like 8G, 8m which needs to be converted into bytes.